Birthday-2024-Project/MainScenes/main_level.tscn

```diff
@@ -1,8 +1,10 @@
-[gd_scene load_steps=13 format=3 uid="uid://bkys348mactw2"]
```

Comment:
I'm not sure about having Exit button on the left, maybe we should have it on the ribbon just like Settings so it will look good right next to each other

Reply:
With the updated exit button art, moved it to the right-side with the settings button